runtime/src/lib.rs

```diff
@@ -255,10 +255,10 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("sora-substrate"),
     impl_name: create_runtime_str!("sora-substrate"),
     authoring_version: 1,
-    spec_version: 101,
+    spec_version: 102,
     impl_version: 1,
     apis: RUNTIME_API_VERSIONS,
-    transaction_version: 101,
+    transaction_version: 102,
```

Comment:
I suggest to move it to the release PR, where node version is updated

Reply:
sorry, done